app/helpers/notifications_helper.rb

```diff
@@ -289,6 +289,14 @@ def sidebar_notification_status(status)
     octicon(NOTIFICATION_STATUS_OCTICON[status], height: 16, class: "sidebar-icon #{status}")
   end
 
+  def subject_with_number(notification)
+    if notification.subject_type == 'Issue' || 'PullRequest'
+      "<span class=\"notification-number\">##{notification.subject_number}</span> #{notification.subject_title}".html_safe
```

Comment:
Having the HTML concatenated inline feels kinda bad here, especially as it requires using `.html_save` which makes me worry about XSS attacks.

Perhaps instead, in the view we can just conditionally render the number and it's span?

Reply:
ah! yes, I can fix that up this evening.